src/screens/InputUsername/index.js

```diff
@@ -246,11 +261,21 @@ const ChooseUsername = () => {
         {
           text: 'Add profile picture',
           style: 'cancel',
-          onPress: () => onPhoto()
+          onPress: () => {
+            AnalyticsEventTracking.eventTrack(
+              BetterSocialEventTracking.ONBOARDING_USERNAME_PROFILE_PIC_ALERT_ADD_PHOTO
+            );
+            onPhoto();
+          }
         },
         {
           text: 'Skip',
-          onPress: () => next()
+          onPress: () => {
+            AnalyticsEventTracking.eventTrack(
+              BetterSocialEventTracking.ONBOARDING_USERNAME_PROFILE_PIC_ALERT_SKIP
+            );
+            next();
+          }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The event tracking code is repeated multiple times in this hunk. To improve maintainability and readability, consider creating a separate function for event tracking that takes the event type as an argument.

```diff
+  const trackEvent = (eventType) => {
+    AnalyticsEventTracking.eventTrack(eventType);
+  }

  {
    text: 'Add profile picture',
    style: 'cancel',
    onPress: () => {
-     AnalyticsEventTracking.eventTrack(
-       BetterSocialEventTracking.ONBOARDING_USERNAME_PROFILE_PIC_ALERT_ADD_PHOTO
-     );
+     trackEvent(BetterSocialEventTracking.ONBOARDING_USERNAME_PROFILE_PIC_ALERT_ADD_PHOTO);
      onPhoto();
    }
  },
  {
    text: 'Skip',
    onPress: () => {
-     AnalyticsEventTracking.eventTrack(
-       BetterSocialEventTracking.ONBOARDING_USERNAME_PROFILE_PIC_ALERT_SKIP
-     );
+     trackEvent(BetterSocialEventTracking.ONBOARDING_USERNAME_PROFILE_PIC_ALERT_SKIP);
      next();
    }
  }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->